autotest-external.sh

```diff
@@ -31,16 +31,21 @@ function print_syntax {
 }
 
 if ! [ -x "$PHPUNIT" ]; then
-	echo "phpunit executable not found, please install phpunit version >= 4.8" >&2
-	exit 3
+	echo "phpunit executable not found, trying local one from build/integration" >&2
+	if [ -x "$PWD/build/integration/vendor/phpunit/phpunit/phpunit" ]; then
+		PHPUNIT="$PWD/build/integration/vendor/phpunit/phpunit/phpunit"
+	else
+		echo "phpunit executable not found, please install phpunit version >= 9.0" >&2
+		exit 3
+	fi
 fi
 
 PHPUNIT_VERSION=$("$PHPUNIT" --version | cut -d" " -f2)
 PHPUNIT_MAJOR_VERSION=$(echo $PHPUNIT_VERSION | cut -d"." -f1)
 PHPUNIT_MINOR_VERSION=$(echo $PHPUNIT_VERSION | cut -d"." -f2)
 
 if ! [ $PHPUNIT_MAJOR_VERSION -gt 9 -o \( $PHPUNIT_MAJOR_VERSION -eq 9 -a $PHPUNIT_MINOR_VERSION -ge 0 \) ]; then
-	echo "phpunit version >= 9.0 required. Version found: $PHPUNIT_VERSION" >&2
+	echo "phpunit version >= 9.6 required. Version found: $PHPUNIT_VERSION" >&2
```

Comment:
```suggestion
	echo "phpunit version >= 9.0 required. Version found: $PHPUNIT_VERSION" >&2
```

Or you adjust the if condition above and the install hint in line 38

Reply:
will not work with drone that only provides 9.4.4